src/libxdata/parsers/BaseParser.cpp

```diff
@@ -42,10 +42,10 @@ std::string BaseParser::parseHeader() {
     getLine(line);
 
     std::istringstream lineStr(line);
-    int version;
+    char origin;

```

Comment:
Not sure if I understand this change - isn't origin the A or I that's parsed above? And the version only on a single line? Source for example here: https://developer.x-plane.com/article/flightplan-files-v11-fms-file-format/